Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -233,6 +236,7 @@ struct AssessmentView: View {
             Button("Yes") {
                 Task {
                     if let pId = vm.submission?.participation.id {
+                        await vm.notifyThemisML(participationId: pId, exerciseId: exercise.id)
```

Comment:
I think we should notify after we sent the assessment, otherwise the ThemisML server might not get the assessment and crash

Reply:
true